chef-provisioning-docker.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.homepage = 'https://github.com/opscode/chef-provisioning-docker'
 
   s.add_dependency 'chef'
-  s.add_dependency 'chef-provisioning', '~> 0.9'
+  s.add_dependency 'chef-provisioning', '~> 1.0'
```

Comment:
Also, there should NOT be breaking changes between chef-provisioning 1.0 and 1.2 (or 1.0 and any 1.x).  Only new features should be added in a minor release.  If we're making breaking changes within the same major release, we're doing it wrong.  We should file a bug and make sure the change introduced is backwards compatible with all 1.x releases.